tests/ci/ci_config.py

```diff
@@ -278,6 +278,7 @@ class JobConfig:
             "./packages",
             "./docker/packager/packager",
             "./rust",
+            "./tests/ci/version_helper.py",
```

Comment:
is it used in cmake configuration?

Reply:
Correct. This file updates both `cmake/autogenerated_versions.txt` and `src/Storages/System/StorageSystemContributors.generated.cpp` files